generators/server/templates/src/main/java/package/config/LiquibaseConfiguration.java.ejs

```diff
@@ -24,7 +24,11 @@ import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.boot.autoconfigure.liquibase.LiquibaseProperties;
+<%_ if (databaseType === 'sql') { _%>
```

Comment:
there's always going to be SQL here, it only works for SQL databases

Reply:
yeah, I just took old code and didn't paid much attention here. I noted that you removed, so, we should be good to go